docs/blog/2019-06-25-profitable-agency-wordpress/index.md

```diff
@@ -9,7 +9,7 @@ tags:
   - aws
 ---
 
-import { Link } from "gatsby"
+import { Link as GatsbyLink } from "gatsby"
```

Comment:
This change is made to get around for this issue https://github.com/prettier/prettier/issues/7548.